spec/factories/topic_category.rb

```diff
@@ -1,6 +1,6 @@
 FactoryBot.define do
   factory :topic_category do
-    name { Faker::Lorem.word }
+    name { Faker::Lorem.unique.word }
```

Comment:
@mahesh-krishnakumar I didn't know this existed! This is cool!

However, this feature's documentation says that the list of stores uniques should be reset between _"runs"_ to avoid an `Faker::UniqueGenerator::RetryLimitExceeded`.

```ruby
# rails_helper.rb
RSpec.configure do |config|
  # ...
  config.before(:each) do
    Faker::UniqueGenerator.clear
  end
  # ...
end
```